installers/gcp-marketplace/Dockerfile

```diff
@@ -13,11 +13,11 @@ RUN install -D /bin/create_manifests.sh /opt/postgres-operator/cloud-marketplace
 RUN if [ -f /etc/os-release ] && [ debian = "$(. /etc/os-release; echo $ID)" ] && [ 10 -ge "$(. /etc/os-release; echo $VERSION_ID)" ]; then \
```

Comment:
The link above here is partially broken. It looks like the anchor has changed to:

```
https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-on-debian
```